### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by MariamOsman11**
# What I have done
configure linters
-Create branches with different base branches.
-create html and css files
-Open pull requests in GitHub.
-Write descriptive, easy to understand commit messages.
-Write short, detailed, easy to understand descriptions of pull requests.
-Write a descriptive and easy to understand README in business English for every project submitted.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (taldr27) [inline .gitignore:1]: - [ ] Great work on updating your `.gitignore` file to include the `node_modules` directory! However, I wanted to kindly remind you one of the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you also need to create and add a `test.md` file to your `.gitignore`.  By doing this you will fulfill all the requested requirements of the project.🎯
image

File: README.md [hunk 2]
```diff
@@ -1 +1,189 @@
...
+  Where can I download node JS for installation?
+
+  - [Answer_2]
+  Node Js can be downloaded here- https://nodejs.org/en/download/
+  <!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
```
Comment (taldr27) [inline README.md:188]: - [ ] Excellent work on creating a comprehensive and polished `Readme` file for your repository! I do have a suggestion that I believe could take your repository's professionalism to the next level: consider adding a `LICENSE` file to your repository. 💯

  _You can learn how to do it by reading **[this useful link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)**._

**Review by taldr27 — CHANGES_REQUESTED** [review]
## Hello @MariamOsman11, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional and detailed Readme. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by joel-kalema — APPROVED** [review]
Hi @MariamOsman11 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**